.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,26 +1,58 @@
-<!--
- Before submitting a PR please ensure all code is commented and all tests are passing. Make sure to review the changes and ensure that only required changes are included (eg. no unnecessary reformatting by your editor)
--->
+<!---
 
-<!-- Please begin the title of this PR with a list of the packages touched (eg. "cmd, rpc: Add Literal Bells and Whistles") -->
+PLEASE READ CAREFULLY
 
+-->
 
-<!-- Brief but specific list of changes made, describe the change in functionality rather than the change in code -->
 ## Changes
-- 
-- 
-- 
 
-<!-- Details on how to run only the tests for the changes in the PR -->
+<!--
+
+Please provide a brief but specific list of changes made, describe the change
+in functionality rather than the change in code.
```

Comment:
grammar: "made, describing the changes in functionality rather than the changes in code"